index.js

```diff
@@ -35,7 +35,7 @@ app.get('/tweet/:tweetId/:address', (req, res) => {
 
   const {tweetId, address} = req.params
 
-  if (tweetId && /^\d{18,20}$/.test(tweetId) && tweetId.length < 20 && /0x[0-9a-fA-F]{40}/.test(address)) {
+  if (tweetId && /^\d{18,20}$/.test(tweetId) && tweetId.length < 20 && /^0x[0-9a-fA-F]{40}$/.test(address)) {
```

Comment:
without the `^` and `$` this matches over 40 characters.

Also I think the `tweetId.length < 20` is redundant here

Reply:
You are right, the first check overrules the seconds.

Reply:
You are right. You can remove the `tweetId.length < 20`, it is overruled by the first check.